rholang/src/main/scala/coop/rchain/rholang/interpreter/rholangADT.scala

```diff
@@ -85,12 +138,16 @@ object Par {
   def apply(e: Expr): Par    = new Par(e)
   def apply(m: Match): Par   = new Par(m)
 
-  implicit def fromSend(s: Send): Par       = apply(s)
-  implicit def fromReceive(r: Receive): Par = apply(r)
-  implicit def fromEval(e: Eval): Par       = apply(e)
-  implicit def fromNew(n: New): Par         = apply(n)
-  implicit def fromExpr(e: Expr): Par       = apply(e)
-  implicit def fromMatch(m: Match): Par     = apply(m)
+  def forge: Par   = apply(GPrivate(uuid))
```

Comment:
These look like helper methods for your tests. Please put them in the tests.

Reply:
It's not a helper method for tests. It's a method for creating an unforgeable process.

Reply:
Please remove it. It doesn't belong in Par. At best it belongs in GPrivate.